### PR TITLE
RELATED: RAIL-2652, RAIL-2655, RAIL-2659 example improvements

### PR DESCRIPTION
**Pull request by no23reason**
* RAIL-2652: Fix styles in ComboChart InsightView example
* RAIL-2655: Improve Arithmetic measure examples - inlined the defintions (so that the sample source code is useful on its own), made sure the data is meaningful with no zeroes
* RAIL-2659: Fxed DrillingWithExternalData - re-renders, broken drilling, missing style, messy code

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by no23reason** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/581/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2174/ : SUCCESS in 11m 13s

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureMultiplicationExample.tsx [hunk 1]
```diff
@@ -1,9 +1,16 @@
 // (C) 2007-2019 GoodData Corporation
 import React from "react";
 import { PivotTable } from "@gooddata/sdk-ui-pivot";
+import { newArithmeticMeasure, measureLocalId } from "@gooddata/sdk-model";
 import { Ldm, LdmExt } from "../../ldm";
 
-const measures = [LdmExt.NrRestaurants, LdmExt.averageRestaurantDailyCosts, LdmExt.arithmeticMeasure2];
+const averageStateDailyCosts = newArithmeticMeasure(
+    [measureLocalId(LdmExt.NrRestaurants), measureLocalId(LdmExt.averageRestaurantDailyCosts)],
```
Comment (lupko) [inline examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureMultiplicationExample.tsx:8]: you should be able to omit measureLocalId, just use the measures themselves

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureRatioExample.tsx [hunk 2]
```diff
@@ -1,9 +1,16 @@
 // (C) 2007-2019 GoodData Corporation
 import React from "react";
 import { PivotTable } from "@gooddata/sdk-ui-pivot";
+import { measureLocalId, newArithmeticMeasure } from "@gooddata/sdk-model";
 import { Ldm, LdmExt } from "../../ldm";
 
-const measures = [LdmExt.NrRestaurants, LdmExt.TotalSales2, LdmExt.arithmeticMeasure3];
+const averageRestaurantSales = newArithmeticMeasure(
+    [measureLocalId(LdmExt.TotalSales2), measureLocalId(LdmExt.NrRestaurants)],
```
Comment (lupko) [inline examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureRatioExample.tsx:8]: same here

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
 import { Ldm, LdmExt } from "../../ldm";
 
-const measures = [
-    LdmExt.FranchiseFeesAdRoyalty,
-    LdmExt.FranchiseFeesOngoingRoyalty,
-    LdmExt.arithmeticMeasure4,
-    LdmExt.arithmeticMeasure5,
-];
+const sum = newArithmeticMeasure(
+    [measureLocalId(LdmExt.FranchiseFeesOngoingRoyalty), measureLocalId(LdmExt.FranchiseFeesAdRoyalty)],
```
Comment (lupko) [inline examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx:8]: same here

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+    "sum",
+    (m) => m.format("#,##0").title("$ Ongoing / Ad Royalty Sum"),
+);
+
+const difference = newArithmeticMeasure(
+    [measureLocalId(LdmExt.FranchiseFeesOngoingRoyalty), measureLocalId(LdmExt.FranchiseFeesAdRoyalty)],
```
Comment (lupko) [inline examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx:14]: same here

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureMultiplicationExample.tsx (hunk 1 above)
Comment (no23reason) [reply examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureMultiplicationExample.tsx]: Good catch, removed

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx (hunk 3 above)
Comment (no23reason) [reply examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx]: Good catch, removed

File: examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx (hunk 4 above)
Comment (no23reason) [reply examples/sdk-examples/src/examples/arithmeticMeasures/ArithmeticMeasureSumExample.tsx]: Good catch, removed

**Comment by lupko** [comment]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/301/ : SUCCESS in 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/557/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2179/ : SUCCESS in 8m 46s

**Pull request merged**